appveyor.yml

```diff
@@ -1,18 +1,34 @@
-version: 1.0.{build}
+version: 11.0.{build}
+pull_requests:
+  do_not_increment_build_number: true
 branches:
   only:
   - master
 image: Visual Studio 2017
+dotnet_csproj:
+  patch: true
+  file: src\ProgressOnderwijsUtils\ProgressOnderwijsUtils.csproj
+  version: '{version}'
+  package_version: '{version}'
+  assembly_version: '{version}'
+  file_version: '{version}'
+  informational_version: '{version}'

```

Comment:
Do you need any of this other than `version`?

Reply:
I don't think so; in fact, AppVeyor only patches the elements that are already present. I just copy-pasted it form the GUI.